src/components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom';
+
+export default function Navbar() {
+  return (
+    <>
+      <div className="app">
+        <h1>Bookstore</h1>
+        <nav>
+          <ul className="navContainer">
+            <li className="navItems">
+              <Link to="/">Books</Link>
+            </li>
+            <li>
+              <Link to="/categories">Categories</Link>
```

Comment:
- [ ] [OPTIONAL] React router provides a special type of link called `NavLink` which knows whether or not it is `active`. It makes it easier to apply styles on the currently active link. You can check this [documentation](https://reactrouter.com/en/main/components/nav-link) for more insight into the topic. 😎️

Reply:
Thank you for this good advice:)